webpack.common.config.ts

```diff
@@ -442,17 +442,6 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
-    new HtmlWebpackPlugin({
```

Comment:
Remove `learner_group_editor` from the entry (Around line 96)

Reply:
Done.